kmip/pie/api.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2015 The Johns Hopkins University/Applied Physics Laboratory
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import abc
+import six
+
+
+@six.add_metaclass(abc.ABCMeta)
+class KmipClientAPI:
```

Comment:
I'm not a fan of *API, *Interface, I*, etc. class names. I feel like this is KmipClient and the existing KmipClient should be something like PieKmipClient or HazardousKmipClient to show that it maps from KmipClient interface to older KmipClientProxy.